Source/Views/StackViewController.swift

```diff
@@ -69,4 +75,80 @@ final class StackViewController: UIViewController {
         stackView.addArrangedSubview(childViewController.view)
         childViewController.didMove(toParentViewController: self)
     }
+    
+    /// A method to access the scrollView from outside and scroll to a given target ViewController. 
+    /// If no relativePosition parameter is provided, it will try to center the target.
+    ///
+    /// - Parameters:
+    ///   - target: The ViewController which should scroll into focus
+    ///   - relativePosition: An optional parameter to specify the alignement of the target. Depending of
+    ///     the stackViews' orientation some relativePosition values have different meaning: 
+    ///     beginning - top/left, end - bottom/right
+    public func scrollToViewController(target targetViewController: UIViewController,
+                                       position relativePosition: RelativePosition = .center, animated: Bool = false) {
+        
+        guard self.childViewControllers.contains(targetViewController) else {
+            return
+        }
+        
+        var targetOffset = CGPoint(x: 0, y: 0)
+        let targetFrame = targetViewController.view.frame
+        
+        if stackView.axis == .vertical {
+            // keep current horizontal offset
+            targetOffset.x = -scrollView.contentInset.left
+            let topOffset = targetFrame.origin.y - scrollView.contentInset.top
+            let bottomOffset = targetFrame.origin.y + targetFrame.height - scrollView.bounds.height
```

Comment:
instead of `targetFrame.origin.y + targetFrame.height` you can use `targetFrame.maxY` ;-)